bundles-lib/src/test/java/org/apache/metron/bundles/integration/components/MRComponent.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.metron.bundles.integration.components;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hdfs.MiniDFSCluster;
+import org.apache.hadoop.yarn.conf.YarnConfiguration;
+import org.apache.metron.bundles.integration.InMemoryComponent;
+
+import java.io.IOException;
+
+public class MRComponent implements InMemoryComponent {
```

Comment:
Is it potentially worthwhile to extract the duplicated function into it's own, independent setup?  And then have both the libs depend on that independent package?

At that point, it eliminates duplication without actually creating a dependency on the Metron libs.  As we do move towards the shared library between NiFi and Metron, we'd need that stuff moved anyway, right?